lib/sheldon/helpers.rb

```diff
@@ -0,0 +1,28 @@
+module Helpers
+
+  def locate_brain
+    relative_path = '~/sheldon2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.